src/Middleware/RateLimiting/src/RateLimitingOptions.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// Specifies options for the <see cref="RateLimitingMiddleware"/>.
+/// </summary>
+public class RateLimitingOptions
+{
+    // TODO - Provide a default?
+    private PartitionedRateLimiter<HttpContext>? _limiter;
+
+    /// <summary>
+    /// Gets the <see cref="PartitionedRateLimiter{TResource}"/>
+    /// </summary>
+    public PartitionedRateLimiter<HttpContext>? Limiter
+    {
+        get => _limiter;
+    }
+
+    /// <summary>
+    /// Adds a new rate limiter.
+    /// </summary>
+    /// <param name="limiter">The <see cref="PartitionedRateLimiter{TResource}"/> to be added.</param>
+    public RateLimitingOptions AddLimiter<HttpContext>(PartitionedRateLimiter<Http.HttpContext> limiter)
```

Comment:
This is an odd method, it's not adding, it's setting. Why not just make the Limiter property settable?

Reply:
I wrote this with the full implementation in mind (a user could add multiple `PartitionedRateLimiters` which we'd combine into part of our internal mega-limiter), but you're right that for this minimal implementation a setter makes more sense